chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,786 @@
+is very important for understanding whether field realities match the research design.
+\index{monitoring data}
+Monitoring data is used to understand if the
+assumptions made during the research design corresponds to what is true in reality.
+The most typical example is to make sure that, in an experimental design,
+the treatment was implemented according to your treatment assignment.
+
+The implementation of your treatment often have to be carried by field counterparts
+that have more complex realities than only your research.
+Field work will always be more intricate than a lab environment, 
+and the field counterpart might not be aware that their actions are the implementation of your research.
+Therefore, you must acquire monitoring data that tells you how well the treatment assignment in the field
+corresponds to your intended treatment assignment,
+for nearly all research designs.
+
+Another example of a research design where monitoring data is important
+are regression discontinuity (RD) designs
+where the discontinuity is a cutoff for eligibility of the treatment. 
+For example, 
+let's say your project studies the impact of a program for students that scored under 50\% at a test.
+We might have the exact results of the tests for all students, 
+and therefore know who should be offered the program, 
+however that is not the same as knowing who attended the program. 
+A teacher might offer the program to someone that scored 51\% at the test,
+and someone that scored 49\% at the might decline to participate in the program.
+We should not pass judgment on a teacher that offers a program to a student
+they think can benefit from it, 
+but if that was not inline with our research assumptions,
+then we need to understand how common that was.
+Otherwise the result of our research will not be helpful
+in evaluating the program.
+
+Monitoring data is particularly prone to errors 
+relating to merging with other data set. 
+It is too late to think about how monitoring data will be merged
+with the rest of the data, after the monitoring data is already collected.
+If you send a team to simply record the name of all people attending a training,
+then potentially different spellings of names 
+-- especially when names have to be transcribed from other alphabets to the Latin alphabet --
+will be the biggest source of error in your monitoring activity.
+The time to discuss and document how monitoring data will be merged with precision
+to the rest of your data, is when you are creating your data map.
+The solution is often very simple, it is just a matter of solving it before it is too late.
+An example of a solution is to provide the monitoring teams 
+with lists of the people's names spelled the same way as in your master dataset. 
+Include both the people you expect to attend the training,
+and people that you do not expect to attend,
+but do not tell the monitoring team which person you expect to attend the training.
```

Comment:
Add some wording about how the tracking of observations should be defined before any data collection is made, and kept the same across all round of data collection. This is a good example, but is not unique to monitoring data, and does not link to why this should be done at the data map stage, as opposed to right before monitoring.

Reply:
Moved to https://github.com/worldbank/dime-data-handbook/issues/465